test/test_ukf.cpp

```diff
@@ -57,9 +57,9 @@ class RosUkfPassThrough : public RosUkf
 TEST(UkfTest, Measurements)
 {
   std::vector<double> args;
-  args.push_back(0.001);
-  args.push_back(0);
-  args.push_back(2);
+  args.push_back(::sqrt(3));
```

Comment:
These will return to normal in my next PR, which will also address the Euler angle issues.